operations/package_repository_management/Repair_Blobstore.md

```diff
@@ -0,0 +1,194 @@
+# Repair `Blobstore`
+
+Nexus may encounter an issue where a RPM fails to download due to missing blob data in the `blobstore`.
+
+Configure the `Repair: Reconcile component database from blob store` task in Nexus to cleanup any uploaded data that may be incomplete.
+This is not typically needed, so it is considered to be a repair task.
+
+The example in this procedure is for creating a repair task to reconcile the `csm blobstore` to resolve an issue downloading `https://packages.local/repository/csm-noos/x86_64/cray-site-init-1.32.3-1.x86_64.rpm`.
+
+Nexus logs may contain the following warning.
+
+Example output:
+
+```text
+*UNKNOWN org.sonatype.nexus.transaction.RetryController - Exceeded retry limit: 8/8 (org.sonatype.nexus.repository.storage.MissingBlobException: Blob default@35A948FA-BC6DC537-60B22DA2-11B0B4B9-EC112D49:7d049616-0122-4160-b5b1-ed5366dd790e exists in metadata, but is missing from the blobstore)
+```
+
+See the [Nexus documentation on tasks](https://help.sonatype.com/en/tasks.html) for more information.
+
+- [Prerequisites](#prerequisites)
+- [System domain name](#system-domain-name)
+- [Nexus web URL](#nexus-web-url)
+- [Procedure](#procedure)
+
+## Prerequisites
+
+CSM installation is complete.
+
+## System domain name
+
+The `SYSTEM_DOMAIN_NAME` value found in some of the URLs on this page is expected to be the system's fully qualified domain name (FQDN).
+
+(`ncn-mw#`) The FQDN can be found by running the following command on any Kubernetes NCN.
+
+```bash
+kubectl get secret site-init -n loftsman -o jsonpath='{.data.customizations\.yaml}' | base64 -d | yq r - spec.network.dns.external
+```
+
+Example output:
+
+```text
+system.hpc.amslabs.hpecorp.net
+```
+
+Be sure to modify the example URLs on this page by replacing `SYSTEM_DOMAIN_NAME` with the actual value found using the above command.
+
+## Nexus web URL
+
+Nexus is accessible using a web browser at the following URL: `https://nexus.cmn.SYSTEM_DOMAIN_NAME`
+
+An example of what the resulting URL will look like is: `https://nexus.cmn.eniac.dev.cray.com`.
+
+## Procedure
+
+1. Log in to the Nexus web UI.
+
+    Users will need to log in through the Nexus UI. The account is configured through Keycloak with a role mapping for Nexus authentication.
+    The role needed for administrative permissions is `nx-admin` in the `system-nexus-client` role.
+    Scripts may connect by using a username and password in the request.
+
+    ![Keycloak Adding Permissions](../../img/operations/Keycloak_add_nexus_permission.png "Keycloak Adding Permissions")
+
+    ![Nexus Web UI](../../img/operations/Nexus_Web_UI.png "Nexus Web UI")
```

Comment:
This can be the local admin account as seen on https://github.com/Cray-HPE/docs-csm/blob/release/1.6/operations/package_repository_management/Manage_Repositories_with_Nexus.md#use-the-local-nexus-admin-account